x-pack/plugins/apm/public/components/shared/Links/apm/transaction_detail_link.tsx

```diff
@@ -17,6 +18,12 @@ interface Props extends APMLinkExtendProps {
   transactionType: string;
 }
 
+const persistedFilters: Array<keyof APMQueryParams> = [
+  'latencyAggregationType',
```

Comment:
IMHO it's better if `latencyAggregationType` is just a (required?) prop on TransactionDetailLink. The others are UI filters.

Reply:
`latencyAggregationType` is also a UI filter IMO.

I see that all TransactionDetailLink properties are related to the transaction object where this link points to. And by adding a new property I'd need to get this item from the URL on all components that call TransactionDetailLink. Something that is done automatically by adding a new option on `persistedFilters`.

Reply:
UI filters are the dropdowns on the left. This one is not. Let's not add `latencyAggregationType` to that bucket, as we are moving away from it. We should also avoid implicit inheritance. I'm okay with more work if it is more explicit.

Reply:
These are more "global" than latencyAggregationType, which applies only to two pages (service overview and service transaction overview). But more importantly, this is implicit, and I think we should be explicit. I've seen many bugs related to implicit inheritance.

Reply:
changed here: https://github.com/elastic/kibana/pull/87046/commits/bf8870a96540d6e9ca35551f7667067791669ae9